tasks/load_instackenv.yml

```diff
@@ -49,10 +49,33 @@
     - name: set undercloud hostname
       set_fact:
         undercloud_hostname: "{{ undercloud_host }}"
+        instack_firstnode: "{{ (instackenv_content.nodes[0].pm_addr | replace('mgmt-','') | replace('-drac', '')) }}"
```

Comment:
> we need to give support for other labs also, since our CI is not running on these labs

@masco can you please share the instackenv for those machines . I will add a task for it